bin/dynflow-export

```diff
@@ -0,0 +1,57 @@
+#!/bin/env ruby
+
+require 'optparse'
+require 'dynflow'
+
+utils = ::Dynflow::Utilities::Export.new
+
+OptionParser.new do |opts|
+  opts.banner = 'Usage: dynflow-export [options]'
+
+  opts.on('-h', '--help', 'Prints help') do
+    puts opts
+    puts "\nAll options can be subtituted by environment variables, however CLI arguments take precedence."
+    puts <<-DESC
+    * TASK_SEARCH     : scoped search filter (example: 'id = "d99bb632-2414-4c2b-af9a-af475b2a4723"')
```

Comment:
it's not scoped search, right? What's the expected syntax of search, when not in tasks?